Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -105,18 +115,26 @@ int main()
         // notifications to the sample
         case ExtendedActivationKind::Launch:
         {
-            // Request a WNS Channel URI which can be passed off to an external app to send notifications to.
-            // The WNS Channel URI uniquely identifies this app for this user and device.
-            PushNotificationChannel channel{ RequestChannel() };
-            if (!channel)
+            if (pushNotificationManager.IsSupported())
             {
-                std::cout << "\nThere was an error obtaining the WNS Channel URI" << std::endl;
-
-                if (remoteId == winrt::guid { "00000000-0000-0000-0000-000000000000" })
+                // Request a WNS Channel URI which can be passed off to an external app to send notifications to.
+                // The WNS Channel URI uniquely identifies this app for this user and device.
+                PushNotificationChannel channel{ RequestChannel() };
+                if (!channel)
                 {
-                    std::cout << "\nThe remoteId has not been set. Refer to the readme file accompanying this sample\nfor the instructions on how to obtain and setup a remote id" << std::endl;
+                    std::cout << "\nThere was an error obtaining the WNS Channel URI" << std::endl;
+
+                    if (remoteId == winrt::guid{ "00000000-0000-0000-0000-000000000000" })
+                    {
+                        std::cout << "\nThe remoteId has not been set. Refer to the readme file accompanying this sample\nfor the instructions on how to obtain and setup a remote id" << std::endl;
+                    }
                 }
             }
+            else
+            {
+                // Here, the app should handle the case where push notifications supported.
+                std::cout << "\nPush Notifications aren't supported." << std::endl;
```

Comment:
Same comment as above here:

General guidance for applications that don't support Push is to do either one of the following:

Maintain their own persistent connection with their App Service
OR
Use polling over a scheduled interval to synchronize the client
Can we mention that somewhere in the comment?

Reply:
done